.gitignore

```diff
@@ -5,6 +5,7 @@
 build/
 packages
 .packages
+.idea/
```

Comment:
Please put this in your GLOBAL git ignore!

Reply:
done :)